packages/node_modules/@webex/plugin-meetings/src/reconnection-manager/index.js

```diff
@@ -259,26 +259,32 @@ export default class ReconnectionManager {
         // Reconnect has failed
         LoggerProxy.logger.error('ReconnectionManager:index#reconnect --> Reconnection failed.', reconnectError.message);
         LoggerProxy.logger.info('ReconnectionManager:index#reconnect --> Sending reconnect abort metric.');
-        Metrics.postEvent({
+
+        const reconnectMetric = {
           event: eventType.CALL_ABORTED,
           meeting: this.meeting,
           data: {
             errors: [
               {
-                category: errorObjects.category.media,
+                category: errorObjects.category.expected,
                 errorCode: 2008,
                 fatal: true,
                 name: errorObjects.name.mediaEngine,
                 shownToUser: false
               }
             ]
           }
-        });
+        };
 
         if (reconnectError instanceof NeedsRejoinError) {
+          // send call aborded event with catogery as expected as we are trying to rejoin
+          Metrics.postEvent(reconnectMetric);
+
           return this.rejoinMeeting(reconnectError.wasSharing);
         }
 
+        Metrics.postEvent(reconnectMetric);
+
         throw reconnectError;
       });
```

Comment:
```suggestion
        Metrics.postEvent(reconnectMetric);

        if (reconnectError instanceof NeedsRejoinError) {
          // send call aborded event with catogery as expected as we are trying to rejoin
          return this.rejoinMeeting(reconnectError.wasSharing);
        }

         throw reconnectError;
      });
```

Reply:
You are making the same call within and after the `if` statement. Just call it once before the `if` statement.